.github/workflows/_build-docker.yml

```diff
@@ -13,7 +13,8 @@ jobs:
         image:
           - ubuntu-24.10
           - fedora-35
-          - debian-oldstable
+          # Restore Debian oldstable until bookworm become it
```

Comment:
For me, a windows person, it took me some time to understand this phrase.  Can we add a link where we could check how bookworm is doing?

Reply:
Here https://www.debian.org/releases/

Reply:
I was suggesting adding it to the code itself.   It's a good practice to have TODO with full context in the code.  A short instruction when, and how to do it and why this todo was added. 

For example:
```
Restore this Debian run, when  Bookworm https://www.debian.org/releases/ will have status: `oldstable`
Python 3.10 does not work with current oldstable (Bullseye)
```